repose-aggregator/src/docs/asciidoc/filters/api-validator.adoc

```diff
@@ -1,6 +1,21 @@
 = API Validator Filter
 
-include::../_includes/in-progress.adoc[]
+The API Validation filter validates all requests against a configured https://www.w3.org/Submission/wadl/[Web Application Description Language (WADL)] file.
+For example, if a request to the origin service is missing the tenant ID in the URI, but the origin service requires the tenant ID to be there, then the request will be rejected before it reaches the origin service.
+The API Validation filter can also be configured to validate the content of the request body against an https://www.w3.org/standards/techs/xmlschema[XML Schema Definition (XSD)] or http://json-schema.org/[JSON Schema] referenced as a grammar in the WADL.
+
+[NOTE]
+====
+A working knowledge of WADL's and XSD's/JSON Schemas will make the configuration of this filter much easier.
+Therefore, it is recommended that you read up on these subjects before attempting to use this filter.
+A good tutorial on XSD 1.1, including a comparison of XSD 1.1 and XSD 1.0, is located at:
+
+* http://www.xfront.com/xml-schema-1-1/
```

Comment:
This particular link I'm good either way on, but for consistency I updated it to the doc title, `XML Schema 1.1 Tutorial`.